### ISSUE TITLE
Can't add custom audio filters

### ISSUE THREAD

**Issue by satvikpendem** [iina/iina#2855]
I am trying to add some custom audio filters. This is how I'd do it in mpv:

```sh
mpv --af=scaletempo=stride=8:overlap=1:search=10 --speed=4 test.mp3
```

I can add the `scaletempo=stride` one fine but I am getting an error with the `overlap` or `search` one.

**System and IINA version:**

- macOS 10.15.3
- IINA 1.0.6 Build 113

**Expected behavior:**

Should play back correctly like mpv

**Actual behavior:**

Reports an error: Error occured when setting filters. Please check your parameter format.

<details>
<summary><b>Crash report:</b></summary>

```
```
</details>

<br>

<details>
<summary><b>mpv log:</b></summary>

```
```
</details>

<br>

**Steps to reproduce:**

- [x] MPV does not have this problem.

**How often does this happen?**

**Comment by varenc** [comment]
You're confusing what is a filter and what are parameters to that filter.

In your mpv example, `scaletempo` is the only filter you're using and `stride=8`, `overlap=1`, and `search=10` are parameters of the scaletempo filter.

This will work fine in IINA if you set the filter name to `scaletempo` and the value to `stride=8:overlap=1:search=10`.  Here's a screenshot: 
![image](https://user-images.githubusercontent.com/167585/78861025-935de800-79e8-11ea-8694-e7112076e580.png)

Here's MPV's docs on the scaletempo filter: https://mpv.io/manual/stable/#audio-filters-scaletempo[

(though as a metapoint, I agree IINA often makes it a bit confusing to figure out how to turn mpv features on in IINA.  Would be super handy if IINA showed you what the equivalent mpv command line option would look like for each filter)

**Comment by satvikpendem** [comment]
Ah thanks a lot! I didn't realize they were parameters to the same filter.

**Comment by varenc** [comment]
glad it's working! I struggled with the exact same thing when trying to setup a sound compression filter in IINA awhile back... This inspired me to go add some helpful details to the [IINA wiki page](https://github.com/iina/iina/wiki/Audio-and-Video-Filters) and I used your filter as an example.

It gets even more confusing when using mpv's `lavfi` filter which takes a param containing a set of ffmpeg's filters+params. Though I have found with mpv I can break things up on the CLI to give me a bit more sanity.  Like I would do `--af=scaletempo="stride=8:overlap=1:search=10"` just for readability.
